src/renderer/components/+preferences/add-helm-repo-dialog.tsx

```diff
@@ -71,12 +73,13 @@ export class AddHelmRepoDialog extends React.Component<Props> {
     dialogState.isOpen = false;
   }
 
-  @observable helmRepo : HelmRepo = this.emptyRepo;
+  @observable helmRepo = getEmptyRepo();
```

Comment:
just curious, what is the benefit of introducing this as external function instead of class method?

Reply:
Nothing really, it could have been a private method yes. I just don't think it should be a plain object because then we are just assigning a reference which is a bit confusing IMO.